templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -107,24 +85,38 @@ export default function ChatInput(
       content,
       filename: file.name,
       filesize: file.size,
+      filetype: "csv",
     });
     if (!isSuccess) {
       alert("File already exists in the list.");
     }
   };
 
+  const handleUploadPdfFile = async (file: File) => {
+    const base64 = await readContent(file);
+    await uploadPdf({
+      id: uuidv4(),
+      filename: file.name,
+      filesize: file.size,
+      pdfBase64: base64,
+    });
+  };
+
   const handleUploadFile = async (file: File) => {
+    if (alreadyUploaded) {
+      alert("You can only upload one file at a time.");
+      return;
+    }
     try {
       if (file.type.startsWith("image/")) {
         return await handleUploadImageFile(file);
       }
       if (file.type === "text/csv") {
```

Comment:
```suggestion
if (file.type === "text/csv" || file.type === "application/pdf") {
        return await handleUploadDocumentFile(file);
}
```
Note: i thing the right wording would be Document, not Content for such files

`handleUploadDocumentFile` can pass `file.type` to the `upload` function of `useFile` (then we also don't need `uploadPdf`)